app/lib/primer/forms/dsl/input.rb

```diff
@@ -58,7 +58,6 @@ def initialize(builder:, form:, **system_arguments)
           @input_arguments = system_arguments
           @input_arguments.delete(:id) unless @input_arguments[:id].present?
           @label_arguments = @input_arguments.delete(:label_arguments) || {}
-          @label_arguments[:for] = id if id.present?
```

Comment:
This was calling `def id` below which is the same as `@input_arguments[:id]` curious how this is different than what was already in place